src/wasm.h

```diff
@@ -227,6 +227,15 @@ class Literal {
       default: abort();
     }
   }
+  
+  int32_t geti64Low() {
```

Comment:
How about defining a `JSFriendlyLiteral` (with a better name perhaps ;)), and all the webidl bindings would return that. It would be a subclass that adds these convenience methods.

Alternatively, we can fix this up in JS dynamically (add those methods directly) but that's probably less clean.